<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QTableView" name="playlist_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1021</width>
      <height>371</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="seek_slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>811</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>254</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="media_controls_layout">
     <item>
      <widget class="QPushButton" name="previous_button">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_button">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="media_time_lcd">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>410</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time remaining for the current track, in hexadecimal seconds.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Hex</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="add_files_action"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="add_files_action">
   <property name="text">
    <string>Add files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
